AkismetPlugin.inc.php

```diff
@@ -13,7 +13,7 @@
  */
 
 import('lib.pkp.classes.plugins.GenericPlugin');
-
+use PKP\i18n;
```

Comment:
This doesn't look quite right to me.  `PKP\i18n` is just a namespace, I think, not a class.  I think you mean `use PKP\i18n\PKPLocale;`, so that in the future, when you reference `PKPLocale` without a namespace, PHP knows where to find it.